### ISSUE TITLE
Disregard extra punctuation after a mention

### ISSUE THREAD

**Issue by graue** [jeena/Bungloo#110]
This text:

> I have to disagree with you, ^jeena.net. The taste insult would offend me, while I'd just laugh off being called stupid.

results in:

```
    "mentions": [{ "entity": "https://jeena.net." }], 
```

It should result in:

```
    "mentions": [{ "entity": "https://jeena.net" }], 
```

Likewise if a mention is followed by other punctuation symbols like comma, question mark, exclamation mark, close paren, semicolon, apostrophe, etc.

If you wanted to be 100% semantically correct you could look at what exact characters [RFC 1738](http://www.faqs.org/rfcs/rfc1738.html) allows in a URL. However many of these characters (`.` and `?` for example) _are_ allowed in URLs, it's just that in practice no one's entity is going to end with them.

**Comment by jeena** [comment]
I'm using http://medialize.github.com/URI.js/ for that, and it works for all of the examples but the dot :-/ I will report it to them and hope that they come up with a sollution.

**Comment by jeena** [comment]
Fixed by #124
